include/fmt/chrono.h

```diff
@@ -1493,10 +1494,13 @@ struct chrono_formatter {
   void write_pinf() { std::copy_n("inf", 3, out); }
   void write_ninf() { std::copy_n("-inf", 4, out); }
 
-  void format_localized(const tm& time, char format, char modifier = 0) {
+  template <class Callback, class... Args>
+  void format_tm(const tm& time, Callback cb, Args... args) {
     if (isnan(val)) return write_nan();
-    out = detail::write<char_type>(
-        out, time, get_locale(localized, context.locale()), format, modifier);
+    get_locale loc(localized, context.locale());
+    auto w = tm_writer_type(loc, out, time);
+    (w.*cb)(std::forward<Args>(args)...);
```

Comment:
`forward` is not needed because the argument is passed by value, please remove.

Reply:
Done